companion/src/radiodata.cpp

```diff
@@ -212,6 +212,16 @@ void GeneralSettings::convert(Board::Type before, Board::Type after)
       memcpy(switchName[5], switchName[7], sizeof(switchName[5]));
     }
   }
+
+  if (IS_TARANIS(after)) {
+      unsigned int min_contrast = getCurrentFirmware()->getCapability(MinContrast);
```

Comment:
Wrong indents in this block, 4 spaces instead of 2.  Also doubled in the `if/else` just below.